client/src/components/TabularViewComponent/TabularViewComponent.scss

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+@import "../../theme/theme.scss";
+
+.ms-DetailsList {
+    overflow-x: hidden;
+}
+
+.ms-Viewport  {
```

Comment:
Remove extra space between selector and `{`

Reply:
Done